canonical-websites/snapcraft.io
# Small fixes for s.io

# Summary

Fixes #638 
- Fix empty fields case if error on listinig page
- Remove tests that are calling the real API. Today API was slow this means that nothing can be merged until the API is not back in normal again...

# QA

- `./run`
- http://127.0.0.1:5000/account/snaps/tbmb-bsi-test-2/listing
- Upload an icon >256x256px
- Check if all empty fields are empty and not `None`
